src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java

```diff
@@ -261,6 +277,7 @@ public boolean hasHTMLTags(String text){
     private static final int MAX_LOG_LENGTH = 200;
     private static final int MIN_WEIGHT_TO_SHOW_IN_LOG = 10;
 
+    private static final Pattern DOMAIN_WITHOUT_TLD = Pattern.compile("(www\\.)?([^\\.]+).*");
```

Comment:
In fact no. But the regex used in extractDomainNameWithoutTld accepts the privateTopLevelDomain which doesn't have subdomains in it.